contracts/src/core/AlignedLayerServiceManager.sol

```diff
@@ -130,10 +130,15 @@ contract AlignedLayerServiceManager is
         // (batchMerkleRoot,senderAddress) is signed as a way to verify the batch was right
         bytes32 batchMerkleRoot,
         address senderAddress,
-        NonSignerStakesAndSignature memory nonSignerStakesAndSignature
+        NonSignerStakesAndSignature memory nonSignerStakesAndSignature,
+        uint256 i
     ) external onlyAggregator {
         uint256 initialGasLeft = gasleft();
 
+        if (i < 4) {
```

Comment:
You can modify this `4` to a higher value to test higher retries. Don't forget to re-deploy the contracts.